vllm/attention/ops/triton_flash_attention.py

```diff
@@ -1071,10 +1069,6 @@ def attn_fwd(
                 end_m_idx = (start_m + 1) * BLOCK_M
                 start_m_idx = start_m * BLOCK_M
                 causal_start_idx = seqlen_q - seqlen_k
-                if EIGHT_BIT and not EIGHT_BIT_KV:  # noqa: SIM102
```

Comment:
We should keep this support in the kernel, we just don't want to integrate it through the model definition. But the kernel should still support the fused quantization of the output